rust/src/mqtt/mqtt.rs

```diff
@@ -52,6 +54,7 @@ pub enum MQTTEvent {
     MissingMsgId,
     UnassignedMsgtype,
     MalformedTraffic,
+    FloodedFlushed,
```

Comment:
Should be `TooManyTransactions` as thats what we already have in master with more or less the same meaning.